macros/upload_individual_datasets/upload_test_executions.sql

```diff
@@ -148,3 +148,50 @@
         {{ return("") }}
     {% endif %}
 {% endmacro -%}
+
+
+{% macro trino__get_test_executions_dml_sql(tests) -%}
+    {% if tests != [] %}
+        {% set test_execution_values %}
+        {% for test in tests -%}
+            (
+                '{{ invocation_id }}', {# command_invocation_id #}
+                '{{ test.node.unique_id }}', {# node_id #}
+                TIMESTAMP '{{ run_started_at }}', {# run_started_at #}
+
+                {% set config_full_refresh = test.node.config.full_refresh %}
+                {% if config_full_refresh is none %}
+                    {% set config_full_refresh = flags.FULL_REFRESH %}
+                {% endif %}
+                
+                {{ config_full_refresh }}, {# was_full_refresh #}
+                '{{ test.thread_id }}', {# thread_id #}
+                '{{ test.status }}', {# status #}
+                
+                {% set compile_started_at = (test.timing | selectattr("name", "eq", "compile") | first | default({}))["started_at"] %}
+                {% if compile_started_at %}TIMESTAMP '{{ compile_started_at }}'{% else %}null{% endif %}, {# compile_started_at #}
+                {% set query_completed_at = (test.timing | selectattr("name", "eq", "execute") | first | default({}))["completed_at"] %}
+                {% if query_completed_at %}TIMESTAMP '{{ query_completed_at }}'{% else %}null{% endif %}, {# query_completed_at #}
+                
+                {{ test.execution_time }}, {# total_node_runtime #}
+
+                {% if test.adapter_response.rows_affected is none %}
```

Comment:
I had to alter this line to
```suggestion
                {% if test.adapter_response.rows_affected is none or test.adapter_response is not defined %}
```

to avoid an empty string